### DISCUSSION TITLE
Slack invite link is broken in the README

### DISCUSSION THREAD

**Discussion by redeux** [haraka/Haraka#3045] [General]
**Describe the bug**
The Slack invite link is broken in the README. It just redirects to the main page for the signup service.

**Expected behavior**
A signup page for the Haraka slack server

**Observed behavior**
Main page redirect

**Steps To Reproduce**
Click the [Join us on Slack](https://communityinviter.com/apps/harakaserver/public-inviter) link in the README

**Comment by msimerson** [comment]
I just fired up Slack for the first time in ... um, a while. I scrolled through our channel and 95% of the posts are automated posts from GitHub, every time an issue or PR is created or updated. Issue #2996 makes me think that joining slack is already an issue and likely will remain so. In light of that, should we just change the link to GitHub Discussions?

**Comment by superman20** [comment]
I'd be in favor of just using GitHub Discussions. I like the format and usability better than Slack and I also like having all Haraka info in just 1 place.

**Comment by nogginbox** [comment]
I just found this discussion.

I had already added a [github issue](/haraka/Haraka/issues/3187) about the slack group invite not working on the community page. Discussions seems like a better place than Slack for this sort of stuff.
